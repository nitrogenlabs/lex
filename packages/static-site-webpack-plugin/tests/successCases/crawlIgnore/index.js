module.exports = (locals) => new Promise((resolve) => {
  switch(locals.path) {
    case '/': {
      return resolve(locals.template({
        path: locals.path
      }));
    }
    default: {
      return resolve('404');
    }
  }
});
