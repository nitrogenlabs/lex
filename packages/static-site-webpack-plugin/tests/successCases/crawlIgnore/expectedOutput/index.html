<p>Don't crawl these:</p>
<ul>
  <li><a href="http://webpack.js.org">Absolute Link</a></li>
  <li><a href="//webpack.js.org">Protocol Relative Link</a></li>
  <li><a href="mailto:hello@example.com">Email Link</a></li>
  <li><a href="#hashtown">Hash Link</a></li>
</ul>
